.github/workflows/ci.yml

```diff
@@ -74,8 +74,8 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
       - name: Install dependencies
-        run: yarn install
+        run: yarn install && cd RNFBSDKExample && yarn
```

Comment:
Why do we need to run yarn twice here?

Reply:
One of them is at the top level, one of them is in the example. The example constructed here is less "magical" in that it does not automatically / auto-magically reference the parent directory code. That's a little inconvenient if using the example app for module development - and I could definitely see an improvement in the future that switches the new-style example to again reference the code raw on the filesystem from the parent directory, but in this style the example is a fully-standalone / not-integrated-with-anything react-native app. So that means you need to go into it and yarn install before it will run

Reply:
(note, to implement this change - to have the example again reference the parent directory module code, I think it's just a switch in build.gradle/Podfile/metro.config.js to put the references in place, or alternatively perhaps it can just be an override of the `yarn add` command to be `yarn add ../` or something? But then it will still be referencing *just the module*, not the rest of the modules and thus it will still need a yarn run in the example itself